Engine/DataFeeds/LiveOptionChainProvider.cs

```diff
@@ -27,6 +27,16 @@ namespace QuantConnect.Lean.Engine.DataFeeds
     /// </summary>
     public class LiveOptionChainProvider : IOptionChainProvider
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LiveOptionChainProvider"/> class
+        /// </summary>
+        public LiveOptionChainProvider()
+        {
+            // The OCC website now requires at least TLS 1.1 for API requests.
+            // NET 4.5.2 and below does not enable these more secure protocols by default, so we add them in here
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
```

Comment:
This type of initialization code is typically placed in a static context. We can move it to the static constructor or to the application context (closer to main)